packages/suite/src/views/wallet/coinmarket/common/CoinmarketBalance.tsx

```diff
@@ -56,10 +49,10 @@ export const CoinmarketBalance = ({
     });
 
     if (showOnlyAmount) {
-        if (typeof balance === 'undefined' || isNaN(Number(balance))) return null;
+        if (!balance || isNaN(Number(balance))) return null;
```

Comment:
If the balance is 0, it will return null, but using `balance === undefined` will be fine.